### DISCUSSION TITLE
[TPA3] Comportamiento de las categorias

### DISCUSSION THREAD

**Discussion by eGoicoechea** [dds-jv/foro#148] [TPA - 2020] [answered]
Hola
Les escribo porque estamos con mi grupo debatiendo acerca de la implementacion de las categorias y nos surgio la siguiente duda: 
El comportamiento de las categorias se puede extender a lo siguiente: 

> Bloquear la posibilidad de aceptar nuevos egresos a una determinada entidad, porque la misma superó un monto previamente    establecido.

> Bloquear la posibilidad de agregar entidades base a una entidad jurídica.

> No permitir que una entidad base pueda ser parte de alguna entidad jurídica.

¿Cada categoria puede tener mas de un comportamiento mencionado anteriormente o solo uno? 

Muchas gracias!

**Comment by gastonprieto** [comment, marked as answer]
Buenas!

Puede tener mas de uno, dependiendo de como hayan configurado la categoría.
